src/app/shared/helpers/competency-outcome-analytic-event.ts

```diff
@@ -11,3 +11,14 @@ export const CompetencyOutcomeAnalyticEvent = (faultLevel: CompetencyOutcome): s
       return AnalyticsEvents.REMOVE_DRIVING_FAULT;
   }
 };
+
+export const CompetencyOutcomeGA4Event = (faultLevel: CompetencyOutcome): string => {
```

Comment:
unit test please.  plus can you reduce the duplication and use this enum from fault-comments

enum ValidFaultTypes {
  DRIVING = 'driving',
  SERIOUS = 'serious',
  DANGEROUS = 'dangerous',
}